apps/teams-test-app/e2e-test-data/otherAppStateChange.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "OtherAppStateChange",
```

Comment:
>name [](http://example.com/codeflow?start=3&length=4)

Can you add a hub SDK version restriction to these that the hub SDK must be >= v2.13.1 ? That's obviously not right because your hub SDK changes aren't going to be in v2.13.1 but until we fix the problem where we can't say "latest or greater" we should at least do v2.13.1 then come back and update it once we actually release the version with your changes.

Reply:
@Kangxuan-Ye do I still need to do this? I know you are literally in the process of making changes here right now. Just let me know what the best practice is and I will do it.